From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@zhangmanling__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Lisaco88 committed:
Merge pull request #22809 from dominicbetts/suite-customize-simulate

Add detail to testing article

hegate committed:
Updating content on Suite tutorials

dominicbetts committed:
Address feedback from Hector

Ja-Dunn committed:
Merge pull request #23019 from dominicbetts/916docreview

916docreview

v-thepet committed:
Merge pull request #22807 from dominicbetts/suite-remove-old-articles

Suite remove old articles

dominicbetts committed:
Merge branch 'release-pcs-v2' into suite-remove-old-articles-2

dominicbetts committed:
Add name an logo change instructions

v-thepet committed:
Merge pull request #23407 from dominicbetts/suite-remove-old-articles-2

Suite remove redundant include files

dominicbetts committed:
Address feedback from Sam

dominicbetts committed:
Remove V2

dominicbetts committed:
Address feedback

dominicbetts committed:
Remove unused article

dominicbetts committed:
Update FAQ

dominicbetts committed:
Fix broken links

dominicbetts committed:
Update ToC (minor)

dominicbetts committed:
Minor update

dominicbetts committed:
Add links to resources in GitHub

dominicbetts committed:
More links

dominicbetts committed:
Update link

dominicbetts committed:
Respond to feedback

dominicbetts committed:
Address feedback from doc bash

PRMerger15 committed:
Merge pull request #23941 from dominicbetts/suite-friday-feedback

Address feedback from doc bash

dominicbetts committed:
Add Visual Studio screenshots

dominicbetts committed:
Fix typo

GitHubber17 committed:
Merge pull request #23426 from dominicbetts/suite-logo-customize

Add logo and name change steps

ktoliver committed:
Merge pull request #23638 from dominicbetts/suite-faq-updates

Update FAQ

GitHubber17 committed:
Merge pull request #23636 from dominicbetts/suite-remove-cli

Remove unused article

dominicbetts committed:
Add some screenshots to explore

dominicbetts committed:
Fix merge conflict

Lisaco88 committed:
Merge pull request #23942 from dominicbetts/suite-vs-screenshots

Add Visual Studio screenshots

Lisaco88 committed:
Merge pull request #23634 from dominicbetts/suite-sam-feedback

Address feedback from Sam G

AyoOlubeko committed:
Removing link to query editor

AyoOlubeko committed:
Adding T-SQL editor doc to table of contents

rmca14 committed:
Merge pull request #23773 from dominicbetts/suite-add-repo-links

Add links to resources in GitHub

AyoOlubeko committed:
Adding note about sql datawarehouse capability of query editor

AyoOlubeko committed:
Title edit

AyoOlubeko committed:
Adding note about permissions required to run.

AyoOlubeko committed:
Adding reference to portal query editor tutorial

AyoOlubeko committed:
Adding reference to azure sql editor tutorial

dominicbetts committed:
Remove some content not part of public preview

dominicbetts committed:
Remove additional non public preview content

PRMerger20 committed:
Merge pull request #24085 from dominicbetts/suite-remove-ga-content

Remove some content not part of public preview

PRMerger10 committed:
Merge pull request #24086 from dominicbetts/suite-remove-ga-content

Remove additional non public preview content

dominicbetts committed:
Tidy up

Commit:
Merge pull request #24088 from dominicbetts/suite-remove-ga-content

Tidy up

dominicbetts committed:
Merge branch 'release-pcs-v2' into suite-explore-screenshots

dominicbetts committed:
Add initial set of screenshots to Explore article

dominicbetts committed:
Fix typo

dominicbetts committed:
Add first screenshots

dominicbetts committed:
Add more how to screenshots

v-anpasi committed:
fixed merge conflicts per Dominic

dominicbetts committed:
Merge branch 'release-pcs-v2' into suite-explore-screenshots

dominicbetts committed:
Merge branch 'release-pcs-v2' into suite-howto-screenshots

Ja-Dunn committed:
Merge pull request #24098 from dominicbetts/suite-howto-screenshots

Add screenshots to how to articles

Lisaco88 committed:
Merge pull request #24092 from dominicbetts/suite-explore-screenshots

Add screenshots to Explore article

AyoOlubeko committed:
Shortening active directory authentication h2

dominicbetts committed:
Address feedback

dominicbetts committed:
Fix links

PRMerger15 committed:
Merge pull request #24195 from dominicbetts/suite-address-feedback-overview

Address feedback for overview articles

dominicbetts committed:
Apply feedback recommendations

PRMerger13 committed:
Merge pull request #24198 from dominicbetts/suite-address-feedback-howto

Apply feedback recommendations

dominicbetts committed:
Add more Explore article screenshots

dominicbetts committed:
Add automate screenshots

dominicbetts committed:
Add physical screenshots

dominicbetts committed:
Add physical screenshots

dominicbetts committed:
Tidy up

dominicbetts committed:
Add screenshots to manage article

dominicbetts committed:
Add Maintenance screenshots

dominicbetts committed:
Add Test screenshots

v-shils committed:
Merge pull request #24203 from dominicbetts/suite-screenshots-explore

Add more screenshots

dominicbetts committed:
Update directory management instructions

dominicbetts committed:
Fix link

dominicbetts committed:
Disable warning message

AyoOlubeko committed:
Updating Azure portal entry to be sentence case

AyoOlubeko committed:
Small tweaks to case of query editor

v-anpasi committed:
Merge pull request #24434 from MicrosoftDocs/master

merge from master

dominicbetts committed:
Update instructions for testing with deployed solution

PRMerger16 committed:
Merge pull request #24438 from dominicbetts/suite-fix-code-samples

Disable warning message

Commit:
Merge pull request #24439 from dominicbetts/suite-permissions

Suite permissions

PRMerger20 committed:
Merge pull request #24440 from dominicbetts/suite-update-deploy

Update instructions for testing with deployed solution

AyoOlubeko committed:
Initial draft of Wingtip georestore tutorial

AyoOlubeko committed:
Revert "Initial draft of Wingtip georestore tutorial"

This reverts commit a759b1df2dab8a07a65fbdf2b0e6a1ca92a05699.

AyoOlubeko committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Update IoT Central example

PRMerger20 committed:
Merge pull request #25193 from dominicbetts/suite-update-options

Update IoT Central example in Options article

v-anpasi committed:
Bringing even with master.

Commit:
edit pass: azure-stack-provider-resource-api

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Add CLI article

rjagiewich committed:
Merge pull request #25391 from dominicbetts/suite-v2-cli

Add CLI article

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Update Options article and title

dominicbetts committed:
Initial commit of Raspberry Pi tutorials

dominicbetts committed:
Update C sample

rjagiewich committed:
Merge pull request #25591 from dominicbetts/suite-options

Update Options article and title

AyoOlubeko committed:
Updating h2 headers to prevent word wrap.

Additionally modifying 'next steps' to only have 1 step.

AyoOlubeko committed:
Merge branch 'master' of https://github.com/AyoOlubeko/azure-docs-pr

AyoOlubeko committed:
Modifying screenshots in docs

AyoOlubeko committed:
Updating aad image to have blue border

dominicbetts committed:
Apply feedback from review

PRMerger20 committed:
Merge pull request #25694 from dominicbetts/suite-cli

Apply feedback from CLI review

dominicbetts committed:
Fix merge conflicts

dominicbetts committed:
Merge branch 'master' into release-pcs-v2-fix-merge

Commit:
First Java quickstart.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
Spelling correction.

dominicbetts committed:
Add services

Lisaco88 committed:
Merge pull request #25696 from dominicbetts/release-pcs-v2-fix-merge

Release pcs v2 fix merge conflict in ToC

Lisaco88 committed:
Merge pull request #25595 from dominicbetts/suite-pi

Initial commit of Raspberry Pi tutorials

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Bringing even with master.

Commit:
George Feedback.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
ASA UDA first draft.

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Address feedback

Ja-Dunn committed:
Merge pull request #26129 from dominicbetts/suite-pcs-feedback

Address feedback

Commit:
ASA add topic for JavaScript UDA

Commit:
ASA fix typo in UDA file name.

Commit:
ASA fix typo in UDA doc

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Add device id instructions

Commit:
Including Ruth's feedback

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

rjagiewich committed:
Merge pull request #26212 from dominicbetts/suite-find-connection-string

Add device id instructions

Commit:
ASA fix typo in UDA doc

v-anpasi committed:
Bringing even with master.

dominicbetts committed:
Update for pcs v2

dominicbetts committed:
FAQ updates

mmacy committed:
[acr] add preview tag for geo-rep tutorial

mmacy committed:
[acr] add IMPORTANT note to first tutorial

dominicbetts committed:
Remove duplicate

Commit:
training content

Commit:
training content - acrolynx edits

Commit:
training content - spelling corrrection in toc

ktoliver committed:
Merge pull request #26310 from dominicbetts/pcs-v2-permissions

Pcs v2 permissions

AyoOlubeko committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

dominicbetts committed:
Minor FAQ update

Commit:
Merge pull request #26374 from dominicbetts/pcs-v2-permissions

Minor FAQ update

v-anpasi committed:
Bringing even with master.

Jiaj-Li committed:
improve the link and Next step session

Commit:
Updated my article with Tamra's changes. Also updated .Net article.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
new title and url

Commit:
Minor correction.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
DevOps topic added

Commit:
DevOps topic - fixing links

Commit:
TOC and other minor edits

Commit:
fixing broken link

v-anpasi committed:
Bringing even with master.

rjagiewich committed:
active-directory-saas-shmoopforschools-tutorial

rjagiewich committed:
active-directory-saas-shmoopforschools-tutorial

rjagiewich committed:
active-directory-saas-shmoopforschools-tutorial

AyoOlubeko committed:
Updating tools menu screenshot

Commit:
FIxing bullet caps.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

rjagiewich committed:
active-directory-saas-shmoopforschools-tutorial

CarlRabeler committed:
e-g

rjagiewich committed:
active-directory-saas-shmoopforschools-tutorial

cychua committed:
Update release date for 649

CarlRabeler committed:
acro

cychua committed:
Clarification on MVP extensions

CarlRabeler committed:
acro

CarlRabeler committed:
acro

MicrosoftGuyJFlo committed:
Updates to docs per request from PG

MicrosoftGuyJFlo committed:
Updates to docs per request from PG

Commit:
ASA UDA fix metadata and other minor updates

CarlRabeler committed:
acro

mawah committed:
Add check on provider registration status

Recommended expressly by Alex Sutton

neilpeterson committed:
removed try-it

neilpeterson committed:
updates

JasonWHowell committed:
Update howto-restore-server-cli.md

mlearned committed:
tweaks to remove bug for deployment groups section

CarlRabeler committed:
Update sql-database-geo-replication-overview.md

rafats committed:
Changeing the URL

rafats committed:
Merge branch 'master' of https://github.com/rafats/azure-docs-pr

PRMerger16 committed:
Merge pull request #26533 from mawah/patch-4

Add check on provider registration status

mlearned committed:
tweaks to the notes section

CarlRabeler committed:
ja-dunn

GitHubber17 committed:
Merge pull request #26169 from minhe-msft/master

ASA adding a topic for JavaScript UDA

Commit:
Merge pull request #26537 from CarlRabeler/patch-357

Update sql-database-geo-replication-overview.md

PRMerger14 committed:
Merge pull request #26539 from rafats/master

Fixed the URL

mlearned committed:
verbiage tweaks

Commit:
Merge pull request #26534 from neilpeterson/remove-try-it-aks

removed try-it

HeidiSteen committed:
typo

ktoliver committed:
Merge pull request #26523 from MicrosoftGuyJFlo/SSPRUpdate

Updates to docs per request from PG

Commit:
copy edits and added final section

cynthn committed:
Update index.yml

ktoliver committed:
Merge pull request #19156 from AyoOlubeko/master

Adding tutorial for T-SQL editor

cynthn committed:
Update index.yml

cynthn committed:
Update index.yml

BryanLa committed:
Change Windows typo to Linux

cynthn committed:
Update index.yml

ktoliver committed:
Merge pull request #26322 from mmacy/acr-geo-rep-preview-tags

[acr] add preview tag for geo-rep tutorial

ShawnJackson committed:
Merge pull request #25289 from LoriMerrick/azure-stack-provider-resource-api2

edit pass: azure-stack-provider-resource-api2

PRMerger17 committed:
Merge pull request #26538 from mlearned/master

tweaks to remove bug for deployment groups section

ShawnJackson committed:
Update active-directory-saas-shmoopforschools-tutorial.md

ShawnJackson committed:
Update active-directory-saas-shmoopforschools-tutorial.md

PRMerger14 committed:
Merge pull request #26550 from BryanLa/patch-13

Change Windows typo to Linux

PRMerger18 committed:
Merge pull request #26536 from JasonWHowell/patch-8

Update howto-restore-server-cli.md

Commit:
Removing windows specific references.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Commit:
Merge pull request #26547 from cychua/master

Update 649 release date

ShawnJackson committed:
Merge pull request #26498 from rjagiewich/active-directory-saas-shmoopforschools-tutorial

Edit pass: Active directory saas shmoopforschools tutorial

Jiaj-Li committed:
OK, I removed the en-us/ in the new URLs and also some old URLs.

Jiaj-Li committed:
add en-us in some old URLs again sicne Publish failed

mahesh-unnikrishnan committed:
Update active-directory-ds-mismatched-tenant-error.md

mahesh-unnikrishnan committed:
keep acrolinx happy

ktoliver committed:
Merge pull request #26552 from cynthn/master

Fixing link

ktoliver committed:
Merge pull request #26503 from CarlRabeler/workload-sql-2

e-g

PRMerger18 committed:
Merge pull request #26382 from Jiaj-Li/storageexplorer

improve the link and Next step session

ktoliver committed:
Merge pull request #26339 from bradsev/tdsp-buck-woody-content

training content

PRMerger20 committed:
Merge pull request #26556 from mahesh-unnikrishnan/patch-15

Self-service resolution of mismatched tenant error.

tynevi committed:
Merge pull request #24217 from MicrosoftDocs/release-pcs-v2

Release pcs v2

tynevi committed:
Merge pull request #25800 from roygara/javaStorageQS

First Java storage quickstart

gsacavdm committed:
Remove govfeedback hyperlink

Commit:
Merge pull request #26559 from gsacavdm/patch-2

Remove govfeedback hyperlink

mdshihab committed:
(ForumsTeam) Grammar edit

Just a minor grammar error in the sentence ". . If its has this Azure region mapping, that Azure region’s latency metrics will be used when creating the Traffic View output. . ." has been edited.

mdshihab committed:
(ForumsTeam) Grammar edit

Made a minor change to the sentence ". . . For example, you can specify that running tasks are stopped immediately, stopped immediately and then requeued for execution on another node, or allowed to finish before the node is removed from the pool. . ."

AnoopVasudavan committed:
updating versions

PRMerger16 committed:
Merge pull request #26566 from AnoopVasudavan/master

updating versions

mlearned committed:
tweaks to verbiage, added note for PAT

mlearned committed:
tweaks on style

PRMerger14 committed:
Merge pull request #26570 from mlearned/master

tweaks to verbiage, added note for PAT

Commit:
Fixed duplication error from seconds to days

PRMerger19 committed:
Merge pull request #26546 from HeidiSteen/heidist-master

typo

v-viksur committed:
(ForumsTeam) Requesting to update broken hyperlink 

In this documentation the hyperlink "Windows VM sizes" has broken. requesting to update the hyperlink.

Commit:
Merge pull request #26571 from scottcate/patch-1

Fixed duplication error from seconds to days

BryanApellanes committed:
custom-search: fix id of response

PRMerger13 committed:
Merge pull request #26574 from brapel/custom-search-fix-response

custom-search: fix id of response

hning86 committed:
Update tutorial-classifying-iris-part-2.md

hning86 committed:
Update tutorial-classifying-iris-part-3.md

fhokholdMSFT committed:
Update key-vault-hsm-protected-keys.md

update hashes in private repo

PRMerger14 committed:
Merge pull request #26579 from fhokholdMSFT/patch-5

Update key-vault-hsm-protected-keys.md

Commit:
Merge pull request #26577 from hning86/patch-14

Update tutorial-classifying-iris-part-2.md

PRMerger17 committed:
Merge pull request #26578 from hning86/patch-15

Update tutorial-classifying-iris-part-3.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #26572 from v-viksur/patch-7

(ForumsTeam) Requesting to update broken hyperlink

PRMerger20 committed:
Merge pull request #26562 from mdshihab/patch-2

(ForumsTeam) Grammar edit

PRMerger17 committed:
Merge pull request #26561 from mdshihab/patch-1

(ForumsTeam) Grammar edit

v-alje committed:
Merge pull request #26582 from MicrosoftDocs/FromPublicRepo

From public repo
